src/interfaces/allowlist/IAllowListV2.sol

```diff
@@ -33,6 +33,8 @@ interface IAllowListV2 {
     /// @dev Thrown if an attempt to call `renounceOwnership` is made
     error RenounceOwnershipDisabled();
 
+    function addressEntityIds(address addr) external view returns (EntityId);
```

Comment:
can you add the natspec comments above similar to the other functions

Reply:
https://github.com/superstateinc/ustb/pull/88/commits/5cfd98c86acb5f2ae0104d733b7d110d8330a278